classes/form/FormBuilderVocabulary.inc.php

```diff
@@ -502,6 +505,46 @@ function _smartyFBVTextArea($params, &$smarty) {
 	}
 
 	/**
+	 * Form color input.
+	 * parameters: disabled (optional), name (optional - assigned value of 'id' by default)
+	 * @param $params array
+	 * @param $smarty object
+	 */
+	function _smartyFBVColor($params, &$smarty) {
+		$params['name'] = isset($params['name']) ? $params['name'] : $params['id'];
+		$params['subLabelTranslate'] = isset($params['subLabelTranslate']) ? (boolean) $params['subLabelTranslate'] : true;
+		$params['uniqId'] = uniqid();
+		$smarty->assign('FBV_isPassword', isset($params['password']) ? true : false);
+
+		$colorParams = '';
+		$smarty->clear_assign(array('FBV_disabled', 'FBV_readonly', 'FBV_multilingual', 'FBV_name', 'FBV_value', 'FBV_label_content', 'FBV_uniqId', 'FBV_default'));
+		foreach ($params as $key => $value) {
+			switch ($key) {
+				case 'label': $smarty->assign('FBV_label_content', $this->_smartyFBVSubLabel($params, $smarty)); break;
+				case 'type': break;
+				case 'size': break;
+				case 'inline': break;
+				case 'subLabelTranslate': break;
+				case 'disabled':
+				case 'readonly':
+				case 'name':
+				case 'id':
+				case 'value':
+				case 'uniqId':
+				case 'default':
+					$smarty->assign('FBV_' . $key, $value); break;
+					break;
+				case 'required': if ($value != 'true') $colorParams .= 'required="' + htmlspecialchars($value, ENT_QUOTES, LOCALE_ENCODING) +'"'; break;
+				default: $colorParams .= htmlspecialchars($key, ENT_QUOTES, LOCALE_ENCODING) . '="' . htmlspecialchars($value, ENT_QUOTES, LOCALE_ENCODING). '" ';
+			}
+		}
+
+		$smarty->assign('FBV_textInputParams', $colorParams);
+
+		return $smarty->fetch('form/color.tpl');
```

Comment:
@NateWr, behind your back I'm going to change this to `colour` instead. Be warned. :)